tox.ini

```diff
@@ -1,50 +1,19 @@
 [tox]
-envlist = security,flake8,typing,pylint,black,docs,twinecheck,py38,py39,py310,py311,py312,pypy3.9,pypy3.10
+envlist = pylint,docs,twinecheck,py38,py39,py310,py311,py312,pypy3.9,pypy3.10
 
 [testenv]
 usedevelop = True
 deps =
     -r{toxinidir}/tests/requirements.txt
 commands = py.test --cov=parsel --cov-report=xml {posargs:docs parsel tests}
 
-[testenv:security]
-deps =
-    bandit
-commands =
-    bandit -r -c .bandit.yml {posargs:parsel}
-
-[testenv:flake8]
-deps =
-    {[testenv]deps}
-    flake8==5.0.4
-commands =
-    flake8 {posargs: parsel tests setup.py}
-
-[testenv:typing]
```

Comment:
This one should be kept.

Reply:
Done in https://github.com/scrapy/parsel/pull/287/commits/3d0b49144850b315581baa14f1582da5a2282442